### DISCUSSION TITLE
Undetected chromedriver doesnt change custom UserAgent

### DISCUSSION THREAD

**Discussion by chibuikeogodo** [ultrafunkamsterdam/undetected-chromedriver#1219] [Q&A] [answered]
I have tried changing my custom user agent with UC but it couldn't, I used options.add_argument. Please can anyone help out please

**Comment by rastinder** [comment, marked as answer]
from selenium.webdriver import ChromeOptions
chrome_options = ChromeOptions()
userAgent = ua.random
print(userAgent)
chrome_option.add_argument(f"--user-agent={userAgent}")
driver = Chrome(options=chrome_option)
